pkg/network/ebpf/kprobe_types.go

```diff
@@ -40,3 +42,7 @@ const (
 )
 
 const BatchSize = C.CONN_CLOSED_BATCH_SIZE
+
+var (
```

Comment:
values that aren't tied to C values, should go in `kprobe.go`. This file is not included in the build, but is used to generate `kprobe_types_<os>.go` via the `system-probe.generate-cgo-types` task.